openerp/addons/base/module/wizard/base_import_language.py

```diff
@@ -38,6 +44,9 @@ def import_lang(self, cr, uid, ids, context=None):
             fileobj.seek(0)
     
             tools.trans_load_data(cr, fileobj, fileformat, this.code, lang_name=this.name, context=context)
+        except Exception, e:
```

Comment:
Hello @mge-odoo 
Here I can't find the exception type because raise this exception before raise https://goo.gl/21HFjw . so I keep as it is. thank you